app/views/schedules/new.html.erb

```diff
@@ -4,7 +4,7 @@
 <div class="row">
   <div class="col-md-8 col-md-offset-2">
     <%= form_for(@schedule) do |f| %>
-        <%= render partial: 'fields', locals: {f: f, object: @schedule} %>
+        <%= render partial: 'fields', locals: {f: f, object: @schedule } %>
```

Comment:
If you drop the space, I think this file won't change... or am I missing something?